docker-compose.yml

```diff
@@ -1,8 +1,7 @@
 server:
   build: .
   volumes:
-    - .:/app
-    - /app/node_modules
+    - ./conf.js:/app/conf.js
```

Comment:
@egorbenko I like this lightweight version as the default a bit better, developers will know to use `docker-compose.development.yml` while your "average" user won't intuitively understand how to use `docker-compose.live.yml`. Fully understand where you're coming from though.